docs/recipes/watch-mode.md

```diff
@@ -69,6 +69,10 @@ AVA tracks which source files your test files depend on. If you change such a de
 
 Dependency tracking works for required modules. Custom extensions and transpilers are supported, provided you loaded them using the [`--require` CLI flag] and not from inside your test file. Files accessed using the `fs` module are not tracked.
 
+## Watch mode and the `.only` modifier
+
+You can use the [`.only` modifier] with watch mode. However now when a change is detected, *all* `.only` tests are rerun, even if they're from a test file that is unaffected by the change.
```

Comment:
The `.only` modifier disables watch modes test dependency tracking algorithm. When a change is made, all `.only` tests will be rerun, regardless of whether the test depends on the changed file.